students/dominic_martin/lesson05/prompt_module.py

```diff
@@ -0,0 +1,17 @@
+#!usr/bin/env/Python 3
+
+prompt = "\n".join(("What would you like to do?",
+    "Please choose from the options:",
+    "1 - Send thanks to someone", 
+    "2 - Send thanks to everyone", 
+    "3 - Generate a report", 
+    "4 - Exit",
+    "Enter your response: ")) 
+
+choose_prompt = "\n".join(("Who would you like to send thanks to?",
+    "1 - Ted Bayer", 
+    "2 - Panfila Alvarez", 
+    "3 - JR Reid", 
+    "4 - Simon Laplace",
+    "5 - Jennifer Meyers",  
```

Comment:
this data (the names) cannot be hardcoded, it needs to come from your data dictionary.